site/src/api/api.ts

```diff
@@ -576,6 +576,21 @@ export const updateWorkspaceDormancy = async (
   return response.data;
 };
 
+export const updateWorkspaceAutomaticUpdates = async (
+  workspaceId: string,
+  automaticUpdates: TypesGen.AutomaticUpdates,
+): Promise<void> => {
+  const data: TypesGen.UpdateWorkspaceAutomaticUpdatesRequest = {
```

Comment:
nit: can we call this `req` or something. I think it just feels weird to declare `data` and return something called `data` in the same function.